fundamentos/bloco-4-introducao-a-javascript-e-logica-de-programacao/dia-4-javascript-objetos-e-funcoes/exercicios/funcoes-2.js

```diff
@@ -0,0 +1,17 @@
+// 2 - Crie uma função que receba um arr de inteiros e retorne o índice do maior valor.
+let arr = [2, 3, 6, 7, 10, 1];
+
+function mostValue(arr) {
+
+    let biggerNum = 0; 
+
+    for (let i = 0; i < arr.length; i++) {
+        
+        if (arr[i] > arr[arr.length - 1]) {
```

Comment:
Você precisa sempre conferir com o maior número anterior. Não com o índice anterior do array.
```suggestion
        if (arr[i] > biggerNum) {
```

Em biggerNum, você computa sempre o maior encontrado **até agora**. E ele pode ou não ser o valor que está no índice anterior do array.